tests/parse_tests/program_reduction_tests/if_block_reduction_test.py

```diff
@@ -24,10 +24,10 @@ def test_if_closed_endwhile(self):
     def test_if_closed_endfor(self):
         self.common.status_error("if function() then\n x = 4\n end for")
 
-    def test_if_with_else_if_closed_endfor(self):
+    def test_if_with_else_if_closed_endif(self):
```

Comment:
Not sure about the naming convention `closed`. Only realised what it meant when I saw there are other tests with `endfor` etc. instead of `endif`.